user_tools/src/spark_rapids_pytools/resources/reports/profWrapperReport.yaml

```diff
@@ -22,3 +22,12 @@ reportDefinitions:
     nestedReports:
       - reportId: profCoreOutput
         relativePath: rapids_4_spark_profile
+    tableDefinitions:
```

Comment:
Q: To access the Autotuner results from the core tool on a per-app basis,  we would have to use ProfCoreResultHandler?